### PR TITLE
Small appearance changes

### PR DESCRIPTION
**Pull request by thefeiter**
This is my first try on using github, so if you like you can merge the small changes I made.
Basically it is just adding an Application Icon and a application name to be shown in the taskbar

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
                      "easydict",
                      "lxml",
                      "requests",
-                     "gpxpy"
+                     "gpxpy",
+                     "dataclasses"
```
Comment (UgnilJoZ) [inline setup.py:19]: I wonder if there is a way of preventing installation of this package on Python >= 3.7 – there, dataclasses are in the standard library.

Comment (thefeiter) [reply setup.py:19]: Aah, ok. I was having the issue because I am still on 3.6.9...

Comment (PhilippRigoll) [reply setup.py:19]: > 
> 
> I wonder if there is a way of preventing installation of this package on Python >= 3.7 – there, dataclasses are in the standard library.

Ah, very good idea. I hadn't thougth about it. It is accutaly possible. -> https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies

**Comment by PhilippRigoll** [comment]
Thanks for your contribution! I would love to see your participating.

**Pull request merged**